src/tab.group.ts

```diff
@@ -353,70 +369,52 @@ export default class TabGroup extends LitElement {
       this.isShowOverflowButtons =
         this.#tabListElementRef.value.scrollWidth >
         this.#tabListElementRef.value.clientWidth;
-    }
 
-    if (this.#tabListElementRef.value) {
       this.isDisableOverflowStartButton =
         this.#tabListElementRef.value.scrollLeft <= 0;
-    }
 
-    if (this.#tabListElementRef.value) {
```

Comment:
This same condition was repeated three times. I moved the assignments in all three into a single condition.